doc/guides/debugging/probes.rst

```diff
@@ -0,0 +1,143 @@
+.. _debug-probes:
+
+Debug Probes
+############
+
+Many supported boards in Zephyr include a second microcontroller that serves as
+an onboard debug probe, usb-to-serial adapter, and sometimes a drag-and-drop
+flash programmer. This eliminates the need to purchase an external debug probe
+and provides a variety of debug host tool options.
+
+Several hardware vendors have their own branded onboard debug probe
+implementations: NXP LPC boards have LPC-Link2, NXP Kinetis (former Freescale)
+boards have OpenSDA, and ST boards have ST-LINK. Each onboard debug probe
+microcontroller can support one or more types of firmware that communicate with
+their respective debug host tools. For example, an OpenSDA microcontroller can
```

Comment:
s/their/its/

Reply:
I don't think so. "One or more types of firmware" -> their.

Reply:
Oh, I read the antecedent as "microcontroller", not "types". Makes sense now.